.editorconfig

```diff
@@ -8,3 +8,7 @@ charset = utf-8
 [*.{cpp, hpp, h}]
 indent_style = tab
 indent_size = 4
+
+[*.json]
+indent_style = tab
+indent_size = 2
```

Comment:
Repertoi-e made 2 spaces indents in previous PR. Also JSON files have usually more indentation levels. But we can have 4, makes no difference to me.